### PR TITLE
JBIDE-20236 Batch New and Noteworthy for 4.3.0.Beta2

### PR DESCRIPTION
**Pull request by alexeykazakov**
This is a N&N for features already pushed to master for Beta2. One more feature (QuickFixes) is still in progress - JBIDE-19266. There is a good chance it will be done by Beta2 code freeze. If so then I will create a separate PR (if this one is merged) to update the current N&N.

### CODE REVIEW

File: documentation/whatsnew/batch/batch-news-4.3.0.Beta2.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+= Batch Tools What's New in 4.3.0.Beta2
+:page-layout: whatsnew
+:page-component_id: batch
+:page-component_version: 4.3.0.Beta2
+:page-product_id: jbt_core 
+:page-product_version: 4.3.0.Beta2
+
+== Hyprelinks for @BatchProperty
```
Comment (maxandersen) [inline documentation/whatsnew/batch/batch-news-4.3.0.Beta2.adoc]: spelling error: Hyperlinks

File: documentation/whatsnew/batch/batch-news-4.3.0.Beta2.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+related_jira::JBIDE-19700[]
+
+== Navigation to Batch artifacts from properties editor 
+
+To navigate from Properties editor to the corresponding batch artifact click on the icon at the property name and select Open Artifact.
+
+image::images/4.3.0.Beta2/open.png[]
+
+If there is no such artifact yet then instead of Open Artifact there will be Create Artifact. Click on this menu to open New Batch Artifact wizard.
```
Comment (maxandersen) [inline documentation/whatsnew/batch/batch-news-4.3.0.Beta2.adoc]: This sentence is a bit hard to parse.

suggestion: "If there are no artifact yet then there will be a 'Create artifact' link which when clicked opens 'New Batch Artifact' wizard" ? 

btw. can the link be a bit more specific than "create artifact" ? that is like writing "Create thing" .....is it not prety specific thing there needs to be created ? ie. a job definition ?

Comment (alexeykazakov) [inline documentation/whatsnew/batch/batch-news-4.3.0.Beta2.adoc]: > can the link be a bit more specific than "create artifact" ? 

Yes it can: JBIDE-20258

**Comment by alexeykazakov** [comment]
Replaced by https://github.com/jbosstools/jbosstools-website/pull/434
